### ISSUE TITLE
loss

### ISSUE THREAD

**Issue by 1701000429** [bigmb/Unet-Segmentation-Pytorch-Nest-of-Unets#54]
hello,dice_loss may different from the “edge loss” in the U-net paper。Can you provide the same loss function as the original paper？Looking forward to your reply

**Comment by bigmb** [comment]
I don't know if I have used different loss function than the U-Net paper. 
But for my case (medical Images) , its better to use Dice Loss due to thin boundaries.
Can you let me know what edge loss you are talking about?
